### PR TITLE
Invoice list page fix bugs

### PR DESCRIPTION
**Pull request by Shruti-Apte**
## Notion card
https://www.notion.so/saeloun/Bugs-and-CR-s-for-recently-updated-invoices-section-on-invoice-list-page-9920d158186945b1842f3888be73542e

## Summary

- Fixed the mentioned bugs 

## Preview

Screenshot 2022-11-03 at 12 34 42 PM

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: app/javascript/src/components/Invoices/List/RecentlyUpdated/index.tsx [hunk 1]
```diff
@@ -1,25 +1,44 @@
 import React from "react";
 
+import { currencyFormat } from "helpers";
+import { useNavigate } from "react-router-dom";
```
Comment (prasanthchaduvula) [inline app/javascript/src/components/Invoices/List/RecentlyUpdated/index.tsx:4]: @Shruti-Apte package import should be on top of local imports
```

import { useNavigate } from "react-router-dom";
import { currencyFormat } from "helpers";
```

**Review by prasanthchaduvula — APPROVED** [review]

**Comment by apoorv1316** [comment]
@supriya3105 @Shruti-Apte Please check how the amount of more than 1 lakh is shown. Is it a requirement?
Screenshot 2022-11-04 at 10 23 46 AM

**Comment by Shruti-Apte** [comment]
> @supriya3105 @Shruti-Apte Please check how the amount of more than 1 lakh is shown. Is it a requirement? Screenshot 2022-11-04 at 10 23 46 AM

@apoorv1316 Yes !! Amount greater than 99999 should be displayed with suffix(according to currency L/M) with 2 decimal points ! This is what I discussed with sonam and supriya !  @supriya3105 can you check if this is right ?

**Review by apoorv1316 — APPROVED** [review]

**Comment by supriya3105** [comment]
@Shruti-Apte " On clicking an invoice from recently updated section, the admin/owner should be able to view the invoice. "  

Is this fixed in this PR ?

**Comment by Shruti-Apte** [comment]
> @Shruti-Apte " On clicking an invoice from recently updated section, the admin/owner should be able to view the invoice. "
> 
> Is this fixed in this PR ?

Yes!

**Comment by supriya3105** [comment]
@Shruti-Apte I will approve this PR after the short numbers issue is fixed after discussion with Sonam. @sonamvg

File: app/javascript/src/components/Invoices/List/RecentlyUpdated/index.tsx (hunk 1 above)
Comment (prasanthchaduvula) [inline app/javascript/src/components/Invoices/List/RecentlyUpdated/index.tsx:4]: @Shruti-Apte `useNavigate` should be imported before helpers. Always first priority of importing goes to packages then to local files.

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]
@Shruti-Apte Please add the key to map on recently updated invoices list to avoid the console warnings.

**Review by prasanthchaduvula — APPROVED** [review]

**Pull request merged**